cmd/abapEnvironmentRunATCCheck.go

```diff
@@ -166,19 +171,26 @@ func buildATCCheckBody(ATCConfig ATCconfig) (string, string, error) {
 		}
 		softwareComponentString += "</obj:softwarecomponents>"
 	}
-	return packageString, softwareComponentString, nil
+	return checkVariantString, packageString, softwareComponentString, nil
 }
 
-func parseATCResult(body []byte) error {
+func parseATCResult(body []byte, atcResultFileName string) (err error) {
 	if len(body) == 0 {
 		return fmt.Errorf("Parsing ATC result failed: %w", errors.New("Body is empty, can't parse empty body"))
 	}
 	parsedXML := new(Result)
 	xml.Unmarshal([]byte(body), &parsedXML)
-	err := ioutil.WriteFile("ATCResults.xml", body, 0644)
-	if err == nil {
+	if len(parsedXML.Files) == 0 {
+		log.Entry().Info("There were no results from this run, most likely the checked Software Components are empty or contain no ATC findings")
+	}
+	s := string(body)
+	if strings.Contains(s, "<html>") {
```

Comment:
I'm not sure if this check is sufficient. E.g. an ATC error message could contain "<html>". Is it possible to check for the "<html>" tag at the begging of the string?